WordPress/Classes/Services/BlogService.m

```diff
@@ -191,6 +192,38 @@ - (void)syncOptionsForBlog:(Blog *)blog
                        failure:failure];
 }
 
+- (void)syncSettingsForBlog:(Blog *)blog
+                    success:(void (^)())success
+                    failure:(void (^)(NSError *error))failure
+{
+    id<BlogServiceRemote> remote = [self remoteForBlog:blog];
+    [remote syncSettingsForBlog:blog
+                        success:^(RemoteBlogSettings *settings) {
+                            blog.blogName = settings.name;
+                            blog.blogTagline = settings.desc;
```

Comment:
This should get the blog in context with the `objectID` instead of accessing it directly